src/test/suite/tokenizer.test.ts

```diff
@@ -0,0 +1,58 @@
+import * as assert from "assert";
+import { tokenize } from "../../tokenizer";
+import { flatten, range } from "lodash";
+
+type TestType = [string, string[]][];
```

Comment:
I think I'd do
```suggestion
type TestCase = [string, string[]];
```

And then put the `[]` on the end in the two places you use it